### PR TITLE
changes in services_unbound_advanced.php

### PR DESCRIPTION
**Pull request by fabianfrz**
* translation fixes (for example submit button)
* create some options in a function
* remove some not operators
* make short echo tags look more like a template

### CODE REVIEW

File: src/www/services_unbound_advanced.php [hunk 1]
```diff
@@ -133,64 +133,64 @@
                       <tr>
                         <td><a id="help_for_hideversion" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Hide Version");?></td>
                         <td>
-                          <input name="hideversion" type="checkbox" id="hideversion" value="yes" <?=!empty($pconfig['hideversion']) ? "checked=\"checked\"" : "";?> />
```
Comment (AdSchellevis) [inline src/www/services_unbound_advanced.php]: @fabianfrz I don't mind switching the order, but don't see the big gain (issues slip in very easily). For the future, can you keep the order as it is if the result is good? Thanks in advance.

File: src/www/services_unbound_advanced.php [hunk 2]
```diff
@@ -338,21 +338,21 @@
...
-                            <?=gettext("Select the log verbosity.");?>
+                            <?= gettext("Select the log verbosity.") ?>
                           </div>
                         </td>
                       </tr>
                       <tr>
                         <td></td>
                         <td>
-                          <input type="submit" name="Save" class="btn btn-primary" id="save" value="Save" />
+                          <button type="submit" name="Save" class="btn btn-primary" id="save" value="Save"><?= gettext("Save") ?></button>
```
Comment (AdSchellevis) [inline src/www/services_unbound_advanced.php]: @fabianfrz shouldn't this be a normal input/submit? the submit is part of the <form/> tag.

Comment (fabianfrz) [inline src/www/services_unbound_advanced.php]: Hi @AdSchellevis, the button allows to use a text between the tags so I can use the translation, with an input type=submit this does not work (I would have to write that into the value).

Comment (AdSchellevis) [inline src/www/services_unbound_advanced.php]: @fabianfrz I understand, only thing I'm not sure about if this is compatible with all browsers, have you tested that?

Comment (fabianfrz) [inline src/www/services_unbound_advanced.php]: Works with Firefox and Vivaldi without issues

Comment (AdSchellevis) [inline src/www/services_unbound_advanced.php]: ok, I will test Safari on my end.

**Pull request merged**